index.html

```diff
@@ -44,129 +65,150 @@ <h4 class="date">Published on Feb 27, 2014</h4>
         <section class="side-bar">
             <aside>
                 <img src="images/side-image-01.jpg">
-                <h3>Contributing to Open Source Part II: The Real Way</h3>
-                <small>The Odin Project</small>
-                <small>29,985 views</small>
+                <p><strong>Contributing to Open Source Part II: The Real Way</strong><br>
```

Comment:
Don't use the <strong> tag to make regular text bold, use CSS instead. [The HTML Strong Importance Element (<strong>) indicates that its contents have strong importance, seriousness, or urgency. Browsers typically render the contents in bold type.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong)